apstools/utils/catalog.py

```diff
@@ -169,57 +151,28 @@ def getDefaultCatalog():
     raise ValueError("No catalogs available.")
 
 
-def getDefaultDatabase():
+def getDefaultDatabase() -> Optional[Any]:
     """
     Find the "default" database (has the most recent run).
 
-    Note that here, *database* and *catalog* mean the same.
-
-    This routine looks at all the database instances defined in the
-    current session (console or notebook).  If there is only one or no
-    database instances defined as objects in the current session, the
-    choice is simple.  When there is more than one database instance in
-    the current session, then the one with the most recent run timestamp
-    is selected.  In the case (as happens when starting with a new
-    database) that the current database has **no** runs *and* another
-    database instance is defined in the session *and* that additional
-    database has runs in it (such as the previous database), then the
-    database with the newest run timestamp (and not the newer empty
-    database) will be chosen.
-
-    RETURNS
-
-    object or ``None``:
-        Bluesky database, an instance of ``databroker.catalog``
-
-    (new in release 1.4.0)
+    Returns:
+        Bluesky database instance or None
     """
-    from databroker import Broker
```

Comment:
The databroker import can be very slow.  That's why it was embedded locally rather than moved to the global scope in this module.  Let's keep it here and revert the move.